projectEuler/problem4.js

```diff
@@ -0,0 +1,38 @@
+// prompt: find the largest palindrome made from the product of two 3-digit numbers
+
+function isPalindrome(num) {
+  let numString = num.toString();
+  let reversedNumString = numString.split("").reverse().join("");
+  if (reversedNumString === numString) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+// find largest value in unsorted array
+function largestVal(arr) {
+  let max = 0;
+  for (let i = 0; i < arr.length; i++) {
+    if (arr[i] > max) {
+      max = arr[i];
+    }
+  }
+  return max;
+}
+
+// main function
+function largestPal() {
+  let palArr = [];
```

Comment:
I believe `palArr` can be declared using `const` since it is not re-assigned here. The value changes in the sense that the items within the array change, but we do not reassign the variable to a new object.